lib/connect.js

```diff
@@ -373,10 +450,16 @@ function connect(token, options) {
  *  <code>VideoCodecs</code> property.
  *  See <a href="https://www.twilio.com/docs/api/video/rooms-resource#create-room">
  *  here</a> for more information.
- * @property {LogLevel|LogLevels} [logLevel='warn'] - Set the log verbosity
- *   of logging to console. Passing a {@link LogLevel} string will use the same
+ * @property {LogLevel|LogLevels} [logLevel='warn'] - <code>(deprecated: use [Video.Logger](module-twilio-video.html) instead.
+ *   See [examples](module-twilio-video.html#.connect) for details)</code>
+ *   Set the default log verbosity
+ *   of logging. Passing a {@link LogLevel} string will use the same
  *   level for all components. Pass a {@link LogLevels} to set specific log
  *   levels.
+ * @property {string} [loggerName='twilio-video'] - The name of the logger. Use this name when accessing the logger
+ *   used by the SDK. If you provided a {@link LogLevels} object with different levels via the <code>logLevel</code>
+ *   parameter, you can access the loggers used by the SDK using the naming format <code>loggerName + '-' + component</code>.
```

Comment:
we should remove mention of logLevel parameter from here. If they supplied `loggerName` then they definitely should not supply custom logLevels.
```suggestion
 * @property {string} [loggerName='twilio-video'] - The name of the logger. Use this name when accessing the logger used by the SDK. 
```